web/src/components/FormStateContacts.js

```diff
@@ -1,48 +1,51 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Box, Button, Flex } from 'rebass';
 import { Field, FieldArray, FormSection, reduxForm } from 'redux-form';
 
 import { Input } from './Inputs';
 import SectionHeader from './SectionHeader';
 
 const Contacts = ({ fields, meta: { error, submitFailed } }) => (
-  <Box mt={4}>
+  <div className="mt2">
     {fields.map((contact, idx) => (
-      <Box mb={4} key={idx}>
+      <div key={idx} className="mb2">
         <SectionHeader>Contact #{idx + 1}:</SectionHeader>
-        <Flex wrap mx={-2}>
-          <Box px={2} w={[1, 1 / 3]}>
+        <div className="clearfix mxn1">
+          <div className="col col-12 sm-col-4 px1">
             <Field name={`${contact}.name`} component={Input} label="Name" />
-          </Box>
-          <Box px={2} w={[1, 1 / 3]}>
+          </div>
+          <div className="col col-12 sm-col-4 px1">
```

Comment:
I'm clearly going to have to go read up on basscss.  I'm guessing this indicates a grid layout, with the typical 12 columns, where at the largest breakpoint the div uses all 12 columns but at the smallest breakpoint is uses 4?  (Are there fewer columns at the different breakpoints?